finagle-base-http/src/main/scala/com/twitter/finagle/http/CookieMap.scala

```diff
@@ -128,13 +128,13 @@ class CookieMap private[finagle] (message: Message, cookieCodec: CookieCodec)
     this += ((cookie.name, cookie))
   }
 
-  override def ++=(xs: TraversableOnce[(String, Cookie)]): this.type = {
```

Comment:
This looks like it's replaced with `addCookies`, and thus an API change.

Reply:
in 2.12, ++= comes from the override in `CookieMapVersionSpecific`
in 2.13, ++= comes from `Map` where it is an alias for `addAll` which is overridden in `CookieMapVersionSpecific`